bin/fin

```diff
@@ -2501,13 +2493,18 @@ smb_share_add()
 # Removes Docksal SMB shares
 smb_share_remove ()
 {
-	local drives=$(wmic logicaldisk get drivetype,name | grep 3 | awk '{print $2}' | sed 's/\r//g')
+	local drives
+	if is_wsl; then drives=$(ls /mnt); else drives=$(ls /cygpath); fi
```

Comment:
What if `/mnt` contains unrelated mounts?

Reply:
I was thinking about that and considered that as an edge case that can be addressed if that ever happens. I like how simple and clean it is now, but if you feel a the more complex treatment via using `wmic.exe` is better, I'm fine with that too. Leave/revert?

Reply:
Also, `ls /cygpath` was a typo, should be `ls /cygdrive`. Already updated that.

Reply:
And here's that edge case - https://github.com/docksal/docksal/issues/1059. So, going back to `wmic.exe`